### PR TITLE
Enhancement 'static' PROTO tag

### PR DESCRIPTION
**Pull request by DavidMansolino**
ref to omichel/webots#5213

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: reference/procedural-proto-nodes.md [hunk 1]
```diff
@@ -102,11 +102,20 @@ Using the [lua-gd](http://ittner.github.io/lua-gd) module it is possible to gene
 
 In addition to these fonts, it is possible to add other TrueType fonts file in your `PROJECT_HOME/fonts` directory.
 
+### Optimization
+
+Using procedural PROTOs can greatly increase the lodaing time of your worlds because every procedural PROTO need to be evaluated.
```
Comment (omichel) [inline reference/procedural-proto-nodes.md]: typo: lodaing

File: reference/procedural-proto-nodes.md [hunk 2]
```diff
@@ -102,11 +102,20 @@ Using the [lua-gd](http://ittner.github.io/lua-gd) module it is possible to gene
...
+
+To reduce the number of evaluation you can add the `static` tag as a comment in the PROTO header (i.e. `# tags: static`). If the `static` tag is present, then in a world who uses the same procedural PROTO several times, the PROTO is evaluated only once if all the fields value are the same.
```
Comment (omichel) [inline reference/procedural-proto-nodes.md]: who &rarr; which

Comment (omichel) [inline reference/procedural-proto-nodes.md]: evaluation &rarr; evaluation**s**

File: reference/procedural-proto-nodes.md (hunk 1 above)
Comment (stefaniapedrazzi) [inline reference/procedural-proto-nodes.md]: Not many months ago, we removed all the `PROTOs` plural form and replaced them by plural name like `PROTO nodes`, `PROTO files`, `PROTO models`, etc.

File: reference/procedural-proto-nodes.md [hunk 3]
```diff
@@ -102,11 +102,20 @@ Using the [lua-gd](http://ittner.github.io/lua-gd) module it is possible to gene
...
+> **Note**:
+This tag should not be used if the result of the PROTOs depend of something else than the fields value (e.g. use a random value).
```
Comment (stefaniapedrazzi) [inline reference/procedural-proto-nodes.md]: if the result (...) depend**s**

Is it correct to use the preposition 'of' with the verb 'to depend'?

File: reference/procedural-proto-nodes.md (hunk 2 above)
Comment (stefaniapedrazzi) [inline reference/procedural-proto-nodes.md]: I would write each sentence on a new line in order to reduce the length of the line.

Personally I think that the last sentence is very complicated and quite difficult to understand.
It also contains 2 if conditions.

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**